source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButton.mm

```diff
@@ -21,9 +25,58 @@ + (UIButton* )acr_renderButton:(UIView *)view
                                                alpha:1.0]];
 
     CGSize contentSize = [button.titleLabel intrinsicContentSize];
-    button.frame = CGRectMake(0, 0, contentSize.width, contentSize.height);
-    button.contentEdgeInsets = UIEdgeInsetsMake(5,5,5,5);
-
+    [button setFrame:CGRectMake(0, 0, contentSize.width, contentSize.height)];
+    [button setContentEdgeInsets:UIEdgeInsetsMake(5,5,5,5)];
+    
+    std::shared_ptr<AdaptiveCards::BaseActionElement> action = [acoAction element];
+    
+    if( [iconUrl length] != 0 ){
```

Comment:
nit space

Reply:
spaces ( xxx )

---
In reply to: [179586673](https://github.com/Microsoft/AdaptiveCards/pull/1325#discussion_r179586673) [](ancestors = 179586673)